www/main/controller.js

```diff
@@ -582,7 +582,7 @@ module.exports = {
       mode,
       fromScroll,
     };
-    if (document.body.classList.contains('livefeed')) {
+    if (document.body.classList.contains('live-feed-true')) {
```

Comment:
could we not change this to check the setting instead of a class on the html?

Reply:
@tsingh777 @inderpreetsingh veerjis, updated this if condition. We only save the location in store, so I set its value to false when we turn off live feed. And used the location setting in this if condition.